whois-update/src/main/java/net/ripe/db/whois/update/authentication/credential/ClientCertificateCredentialValidator.java

```diff
@@ -36,7 +36,7 @@ public class ClientCertificateCredentialValidator implements CredentialValidator
     public ClientCertificateCredentialValidator(final RpslObjectDao rpslObjectDao,
                                                 final DateTimeProvider dateTimeProvider,
                                                 final LoggerContext loggerContext,
-                                                final @Value("${client.cert.auth.enabled:false}") boolean enabled) {
+                                                final @Value("${client.auth.enabled:false}") boolean enabled) {
```

Comment:
Remember to also add the new property `client.auth.enabled` to all the Whois property files and add a TODO to remove `client.cert.auth.enabled`

Reply:
yes, I will leave both for now in all the env and put to remove client.cert.auth.enabled in the release notes.
So we can do it when we do the release. I think that TODO is not necessary if we have already that specified in the release notes.
What do you think?

Reply:
OK! That's fine